mne/decoding/transformer.py

```diff
@@ -168,35 +170,34 @@ def __init__(self, info=None):
         self.n_channels = None
         self.n_times = None
 
-    def fit(self, epochs_data, y):
+    def fit(self, epochs, y=None):
         """For each epoch, concatenate data from different channels into a single
         feature vector.
```

Comment:
The fit function doesn't do anything, it is just to make it compatible with sklearn. Can you correct the doc, (as well as the one for `fit_transform`)?